src/Assets/color.tsx

```diff
@@ -0,0 +1,7 @@
+const Colors =
```

Comment:
This one also should be moved to config folder.

Reply:
done sir